DP03_06_Using_User-Supplied_Catalogs.ipynb

```diff
@@ -0,0 +1,686 @@
+{
```

Comment:
<p>Would rephrase that "The TAP service does not accept uploads in a Pandas DataFrame format. Thus..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>